client/pages/Login.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useState, useEffect} from 'react';
+import { KeyboardAvoidingView, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+import { useNavigation } from '@react-navigation/core';
+
+import { firebase, auth, provider, signInWithGoogle } from '../configs/firebase';
+
+const styles = StyleSheet.create({
```

Comment:
I'd prefer to have styles at the bottom of the doc, but it's not that important. Would be nice to have it consistent.